contracts/compose.reth-stubs.yml

```diff
@@ -0,0 +1,58 @@
+services:
+  eth-stub:
+    build:
+      context: .
+      dockerfile_inline: |
+        FROM ghcr.io/paradigmxyz/reth:v1.0.7
+        RUN apt-get update && apt-get install -y wget netcat && rm -rf /var/lib/apt/lists/*
+    ports:
+      - 8545:8545
+      - 8546:8546
+    command:
+        - node
+        - '--datadir=/data'
+        - '--http'
+        - '--http.addr=0.0.0.0'
+        - '--http.port=8545'
+        - '--http.api=all'
+        - '--http.corsdomain=*'
+        - '--ws'
+        - '--ws.addr=0.0.0.0'
+        - '--ws.port=8546'
+        - '--ws.api=all'
+        - '--ws.origins=*'
+        - '--auto-mine'
```

Comment:
didn't we want to set `dev.block-time = 1s` here?
https://mangatafinance.slack.com/archives/G01FB4YHGMT/p1727947361912019?thread_ts=1727862030.716929&cid=G01FB4YHGMT

Is this no longer an issue?

Reply:
I don't understand how... Maybe auto-mine was not enabled when we had encountered the bug earlier...
Let's leave it as it is... If we have a problem again then we'll see...